src/spatial_ref/srs.rs

```diff
@@ -451,6 +451,120 @@ impl SpatialRef {
             None
         }
     }
+
+    /// Get spheroid semi major axis. It returns [`Err`] variant in case of non [`OGRERR_NONE`](https://gdal.org/api/vector_c_api.html#c.OGRERR_NONE) error result from orginal method.
```

Comment:
Adding a blank line between the summary and details helps the rustdoc render more cleanly.

```suggestion
    /// Get spheroid semi major axis. 
    ///
    /// Returns [`Err`] variant in case of non [`OGRERR_NONE`](https://gdal.org/api/vector_c_api.html#c.OGRERR_NONE) error result from the C library.
```